design/cstor/v1/cstorpool.md

```diff
@@ -247,9 +247,28 @@ type CStorPoolInstanceStatus struct {
 
 // CStorPoolInstanceCapacity stores the pool capacity related attributes.
 type CStorPoolInstanceCapacity struct {
+	// Amount of physical data (and its metadata) written to pool
+	// after applying compression, raid properties, etc..,
+	Used resource.Quantity `json:"used"`
+	// Amount of usable space in the pool after excluding
+	// metadata and raid properties
+	Free resource.Quantity `json:"free"`
+	// Sum of usable capacity in all the data raidgroups
 	Total resource.Quantity `json:"total"`
-	Free  resource.Quantity `json:"free"`
-	Used  resource.Quantity `json:"used"`
+	// ZFSCapacityAttributes contains advanced information about pool capacity details
+	ZFS ZFSCapacityAttributes `json:"zfs"`
```

Comment:
The `JSON` tag should not be `zfs`
Let us transform this : 
`zfs` --------> `ZFSCapacityAttributes`